source/chapters/user_interface.rst

```diff
@@ -620,14 +620,24 @@ Headphone and Master Mix Controls
   This knob allows you to adjust the :term:`balance` (left/right orientation) of
   the master output.
 
-**Volume Knob**
-  The Volume Knob controls the overall volume of the master output. Adjust
+**Gain Knob**
+  The Gain Knob controls the overall gain of the master output. Adjust
   this knob so that the :ref:`Master VU meters <interface-vu-meter>` are just
   at the peak.
 
-  .. hint:: If the Peak indicator on top of the Master VU meter flashes, the
-            master output signal is clipping (too loud). Lower the volume with
-            using the volume knob.
+  .. warning:: Gain is not the same as volume. Improperly setting the gain too
+               high or too low will decrease your sound quality. If you want to
+               adjust your overall volume, do so by adjusting the gain as close
+               to the sound output as possible. If you can adjust a PA system's
+               amplifier, adjust it there. This is not always an option for DJs,
+               so the next best place is usually the DJ mixer. If there is no DJ
+               mixer, adjust your soundcard. If your soundcard does not have any
```

Comment:
I would recommended this: Adjust you amplifier to the maximum volume you need during the night, (all other gains at unity, and volume sliders at Max. 
Set the current volume either at a analog DJ Mixer or at the Volume panel of your OS (eg alsamixer). 
If you cannot effort this use the Master gain to reduce the volume.